edk2toolext/environment/version_aggregator.py

```diff
@@ -17,7 +17,7 @@
 class version_aggregator(object):
```

Comment:
what was the reason for this file changes?  Just better structure?  or was their a functional change?

Reply:
I think better structure- making it a little more clear that you're not supposed to touch the Versions variable

Reply:
sounds good.  I think it would be good in future to structure your commit to indicate that.  Helps me as the reviewer and then also lets us decide if ok in this PR.  I value our ability to refactor as we go but we should be clear and at times we all run a little fast/loose. :) 

Overall i am good.  Feel free to complete the PR with a good commit message.

Reply:
No I agree- a better explanation in the commit would have been helpful